pkg/sys/termios.go

```diff
@@ -15,19 +15,16 @@ import (
 // Termios represents terminal attributes.
 type Termios unix.Termios
 
-// NewTermiosFromFd extracts the terminal attribute of the given file
-// descriptor.
-func NewTermiosFromFd(fd int) (*Termios, error) {
-	var term Termios
-	if err := Ioctl(fd, getAttrIOCTL, uintptr(unsafe.Pointer(&term))); err != nil {
-		return nil, err
-	}
-	return &term, nil
+// TermiosForFd returns a pointer to a Termios structure if the file
+// descriptor is open on a terminal device.
+func TermiosForFd(fd int) (*Termios, error) {
+	term, err := unix.IoctlGetTermios(fd, getAttrIOCTL)
+	return (*Termios)(unsafe.Pointer(term)), err
```

Comment:
The conversion to unsafe.Pointer is not needed.

Reply:
Yep, again, leftover from previous changes as I explored various refactorings.